plugins-ui/src/modules/core/components/ui/button/Button.tsx

```
@@ -0,0 +1,38 @@
+import { ReactNode } from "react";
+import "./Button.css";
+
+type ButtonProps = {
+  type: "button" | "submit";
+  styleType: "primary" | "secondary" | "tertiary";
+  size: "mini" | "small" | "medium";
+  children: ReactNode;
+  className?: string;
+  style?: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)